### PR TITLE
Handle offer after packet

### PR DESCRIPTION
**Pull request by michaeldjeffrey**

### CODE REVIEW

**Comment by macpie** [comment]
This looks alright so far

File: rebar.lock [hunk 1]
```diff
@@ -132,6 +132,10 @@
   {git,"https://github.com/helium/libp2p-crypto.git",
        {ref,"a969bb7affc7d3fa1472aef80d736c467d3f6d45"}},
   1},
+ {<<"lorawan">>,
+  {git,"https://github.com/helium/erlang-lorawan.git",
+       {ref,"49925a083347e91e74daee3516a2105e82ef6c07"}},
+  1},
```
Comment (mikev) [inline rebar.lock:138]: I'd recommend the latest - 32d1c580c26ba41ad5230d080d4c0e23cf273d86
In particular the list_to_existing_atom fixes potential memory issues.

**Review by dpezely — DISMISSED** [review]

**Review by dpezely — DISMISSED** [review]
In case we want to preserve the Reputation System since it's being reverted/removed, I've grabbed a copy of:
https://github.com/helium/router/pull/823/commits/616c75d14c2ea3a3845037664283afdfda5cf7ee.diff
(just add `.diff` to the commit's URL)

File: rebar.config [hunk 2]
```diff
@@ -22,7 +22,8 @@
     {httpc_aws, {git, "https://github.com/helium/httpc-aws.git", {branch, "master"}}},
     {websocket_client, "1.4.2"},
     {erlang_v8, {git, "https://github.com/Vagabond/erlang_v8.git", {branch, "master"}}},
-    {blockchain, {git, "https://github.com/helium/blockchain-core.git", {branch, "master"}}},
+    {blockchain,
+        {git, "https://github.com/helium/blockchain-core.git", {branch, "mj/ul-direct-send-2"}}},
```
Comment (michaeldjeffrey) [inline rebar.config:26]: do we want to change this back to master?

Comment (macpie) [reply rebar.config:26]: good catch

**Review by dpezely — DISMISSED** [review]

**Review by dpezely — APPROVED** [review]

**Pull request merged**